src/test/ui/ufcs/ufcs-qpath-self-mismatch.stderr

```diff
@@ -5,6 +5,7 @@ LL |     <i32 as Add<u32>>::add(1, 2);
    |     ^^^^^^^^^^^^^^^^^^^^^^ no implementation for `i32 + u32`
    |
    = help: the trait `std::ops::Add<u32>` is not implemented for `i32`
+   = note: required by `std::ops::Add::add`
```

Comment:
I'll take a look later how it can be done, but it would be better if we didn't add this redundant `note`.